src/utils/graphql/queries/getReviewNew.query.ts

```diff
@@ -3,15 +3,9 @@ import { gql } from '@apollo/client'
 // For review responses linked to applicationResponses, we want to get either submitted responses
 // Or draft response that belong to current user
 export default gql`
-  query getReviewNew($reviewAssignmentId: Int!, $userId: Int!, $sectionIds: [Int!]) {
+  query getReviewResponses($reviewAssignmentId: Int!, $userId: Int!, $sectionIds: [Int!]) {
```

Comment:
ohh I've change the getReviewNEW to only return review response, and made the getReviewInfo to return all the stuff related to review (assignment, review status, etc..).  Otherwise we seemed to be getting information for review in both queries